subprojects/jacoco/src/testFixtures/groovy/org/gradle/testing/jacoco/plugins/fixtures/JacocoCoverage.groovy

```diff
@@ -33,10 +34,16 @@ final class JacocoCoverage {
         jacocoVersion.compareTo(supportedJacocoVersion) == -1
     }.asImmutable()
 
-    final static List<String> SUPPORTS_JDK_8_OR_HIGHER = filter(JacocoVersion.SUPPORTS_JDK_8)
-    final static List<String> SUPPORTS_JDK_9_OR_HIGHER = filter(JacocoVersion.SUPPORTS_JDK_9)
+    private final static List<String> SUPPORTS_JDK_8_OR_HIGHER = filter(JacocoVersion.SUPPORTS_JDK_8)
+    private final static List<String> SUPPORTS_JDK_9_OR_HIGHER = filter(JacocoVersion.SUPPORTS_JDK_9)
+    private final static List<String> SUPPORTS_JDK14 = filter(JacocoVersion.SUPPORTS_JDK_14)
```

Comment:
This is similar to what I was talking about above.  We should strive for a single "ALL" list that gets filtered.  

We should check if the COVERAGE_CHECK_xxx versions make sense.  They may need to be filtered by JDK version too.